### ISSUE TITLE
Reverse order?

### ISSUE THREAD

**Issue by sashabeep** [mnoskov/pagebuilder#91]
For now there is no option to reverse order or sort by block id in snippet parameters

**Comment by mnoskov** [comment]
Because order determined in admin panel.

**Comment by sashabeep** [comment]
@mnoskov yep, but somtimes it would be great to reverse order on the frontend, for example, for the latest works or customer replies, to show latest added first
